sci-electronics/kicad/kicad-4.0.6.ebuild

```diff
@@ -0,0 +1,198 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit cmake-utils eutils flag-o-matic gnome2-utils python-single-r1 wxwidgets vcs-snapshot versionator xdg
+
+DESCRIPTION="Electronic Schematic and PCB design tools."
+HOMEPAGE="http://www.kicad-pcb.org"
+
+SERIES=$(get_version_component_range 1-2)
+
+SRC_URI="https://launchpad.net/${PN}/${SERIES}/${PV}/+download/${P}.tar.xz
+	!minimal? (
+		http://downloads.kicad-pcb.org/libraries/${PN}-footprints-${PV}.tar.gz
+		http://downloads.kicad-pcb.org/libraries/kicad-library-${PV}.tar.gz
+	)
+	i18n? ( https://github.com/KiCad/${PN}-i18n/archive/${PV}.tar.gz -> ${P}-i18n.tar.gz )"
+
+LICENSE="GPL-2+ GPL-3+ Boost-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc examples github i18n libressl minimal +python"
+LANGS="bg ca cs de el es fi fr hu it ja ko nl pl pt ru sk sl sv zh_CN"
+for lang in ${LANGS} ; do
+	IUSE="${IUSE} linguas_${lang}"
```

Comment:
LINGUAS as USE flags is deprecated and half-broken. If you need to control localizations for meaningful things (like fetching additional packages or installing big files), use L10N instead. Otherwise, just remove the logic.